modules/zram/udiskslinuxmanagerzram.c

```diff
@@ -310,6 +310,56 @@ delete_conf_files (GError **error)
   return rval;
 
 }
+
+static UDisksObject **
+wait_for_zram_objects (UDisksDaemon *daemon,
+                       gpointer      user_data)
+{
+  gchar **zram_p = NULL;
+  gint next_obj = 0;
+  gboolean success = TRUE;
+  gint num_zrams = 0;
+  UDisksObject **objects = NULL;
+
+  num_zrams = g_strv_length ((gchar **) user_data);
+  objects = g_new0 (UDisksObject *, num_zrams);
+
+  for (zram_p = (gchar **) user_data; *zram_p != NULL; zram_p++, next_obj++)
```

Comment:
Would be nice to either describe what ``user_data`` contains here or store it in a properly named local variable of type ``gchar **``.